src/com/xilinx/rapidwright/tests/CodePerfTracker.java

```diff
@@ -48,6 +52,8 @@ public class CodePerfTracker {
 
     private Map<String,Long> inflightTimes;
 
+    private List<Pair<Long,Long>> totalPeakMemUsages;
```

Comment:
Since we're no longer tracking the current, do we still need to acquire and store `Pair`?

Reply:
Generally, I disagree with adding dead code that cannot get exercised in normal operation and might even cause confusion down the line for anyone wanting to modify this code.

If you feel strongly, please leave a comment behind, and perhaps rename `totalPeakMemUsages` to `totalOSMemUsages` or similar.

Reply:
Updated so the code is not dead, but can be turned on with a flag.